packages/flutter/lib/src/material/radio.dart

```diff
@@ -93,7 +93,7 @@ class Radio<T> extends StatefulWidget {
 
   /// The color to use when this radio button is selected.
   ///
-  /// Defaults to accent color of the current [Theme].
+  /// Defaults to [Theme.toggleableActiveColor].
```

Comment:
ThemeData

I'll stop repeating this feedback. There are more to fix...

Reply:
Fixed